libraries/AP_Common/missing/cmath

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <toolchain.h>
```

Comment:
this should not be here... add toolchain.h (or a better name... ideally it would be config.h) to the compiler flags as `-include` directive